subsys/net/ip/ipv4_autoconf.c

```diff
@@ -168,12 +168,12 @@ static void ipv4_autoconf_send(struct net_if_ipv4_autoconf *ipv4auto)
 		ipv4auto->announce_cnt = 0U;
 		ipv4auto->conflict_cnt = 0U;
 		(void)memset(&ipv4auto->current_ip, 0, sizeof(struct in_addr));
-		ipv4auto->requested_ip.s4_addr[0] = 169U;
```

Comment:
All of these changes need to be reverted.. The v4 auto-configuration is supposed to generate ip address from `169.254.X.X` range per https://tools.ietf.org/html/rfc3927..

Reply:
Yes. This is also fixed in the other PR.